sdk/openai/Azure.AI.OpenAI/tests/Utils/Config/IConfiguration.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework.Internal;
+
+namespace Azure.AI.OpenAI.Tests.Utils.Config;
+
+/// <summary>
+/// A test configuration for an Azure resource.
+/// </summary>
+public interface IConfiguration
```

Comment:
Would it be possible to make this type internal?

Reply:
Why? This is test code

Reply:
Based on @jsquire comments, I think the better thing long term would be to use the `Microsoft.Extensions.Configuration` framework for configuration longer term anyway.